site/content/includes/installation/install-gateway-api-experimental-features.md

```diff
@@ -0,0 +1,23 @@
+---
+docs: "DOCS-000"
+---
+
+To use Gateway API experimental resources, the Gateway API resources from the experimental channel must be installed before deploying NGINX Gateway Fabric. Additionally, NGINX Gateway Fabric must have experimental features enabled.
+
+{{<caution>}}As noted in the [Gateway API documentation](https://gateway-api.sigs.k8s.io/guides/#install-experimental-channel), future releases of the Gateway API can include breaking changes to experimental resources and fields.{{</caution>}}
+
+To install the Gateway API resources from the experimental channel, run the following:
+
+```shell
+kubectl kustomize "https://github.com/nginxinc/nginx-gateway-fabric/config/crd/gateway-api/experimental?ref=v1.4.0" | kubectl apply -f -
+```
+
```

Comment:
Could be nice to show example output as the consequence of the `kubectl` command.

Reply:
I'm up for it, though do you think it would be beneficial to add it to https://github.com/nginxinc/nginx-gateway-fabric/blob/b8c64df41a709ea84e4ecce46ffb9b91ee0a007a/site/content/includes/installation/install-gateway-api-resources.md?plain=1#L10 also? I don't think it makes sense to do it to one but not the other.

Reply:
Yeah, sounds good to me. It's a pattern (Showing users expected output) we're encouraging adoption of: it should make successful steps clearer for a reader with less experience.

Reply:
Generally, I'm a fan of including output for kubectl commands, but in this case, I don't think we want to. The output is a list of CRDs that are installed for the Gateway API version and channel in the command:

```
customresourcedefinition.apiextensions.k8s.io/gatewayclasses.gateway.networking.k8s.io configured
customresourcedefinition.apiextensions.k8s.io/gateways.gateway.networking.k8s.io configured
customresourcedefinition.apiextensions.k8s.io/grpcroutes.gateway.networking.k8s.io configured
customresourcedefinition.apiextensions.k8s.io/httproutes.gateway.networking.k8s.io configured
customresourcedefinition.apiextensions.k8s.io/referencegrants.gateway.networking.k8s.io configured
```

This list is likely to change with every release of the Gateway API. I don't think the benefit to the user is big enough to warrant maintaining this output.